src/constants/events.ts

```diff
@@ -3,4 +3,6 @@ export const CONNECT_EVENT = 'CONNECT'
 export const TRANSACTION_EVENT = 'TRANSACTION_SUBMITTED'
 export const ATTRIBUTION_EVENT = 'ATTRIBUTION'
 export const REFERRER_EVENT = 'REFERRER'
+// CAUGHT events
 export const FIRST_PAGE_VISIT = 'FIRST_PAGE_VISIT'
+export const CAUGHT_TRANSACTION_EVENT = 'CAUGHT_TRANSACTION_SUBMITTED'
```

Comment:
We can keep using `TRANSACTION_SUBMITTED`

Reply:
Transaction submitted expects to have the transaction hash. Is that right that it will not be necessary anymore? 
Is that okay, that we will have different structures under single event for single customer from data perspective?

Reply:
This is correct, as discussed